res/css/views/right_panel/_ThreadPanel.pcss

```diff
@@ -106,7 +106,7 @@ limitations under the License.
         content-visibility: visible;
     }
 
-    .mx_EventTile,
+    .mx_EventTile[data-layout],
```

Comment:
when is `data-layout` *not* present, and what's the rationale for doing this with an attribute selector rather than a class?